alonzo/impl/src/Cardano/Ledger/Alonzo.hs

```diff
@@ -214,6 +214,7 @@ instance CC.Crypto c => EraModule.SupportsSegWit (AlonzoEra c) where
   fromTxSeq = Alonzo.txSeqTxns
   toTxSeq = Alonzo.TxSeq
   hashTxSeq = Alonzo.hashTxSeq
+  numSegComponents = 3
```

Comment:
Would it not be easier to have this return the total number of segregated components, rather than just the "non-body" ones?

Reply:
I think it would. I wasn't sure if that was breaking the abstraction, though (ie why should `SupportsSegWit` know about block size). but maybe that's silly.

Reply:
oh wait, you mean to not include the block header. yea, that's better!